packages/blocks/google-cloud/src/lib/actions/get-recommendations-cli-action.ts

```diff
@@ -0,0 +1,298 @@
+import { createAction, Property } from '@openops/blocks-framework';
```

Comment:
Okay so the reason this looks do confusing is:

For the get recommendations call the user must provide either 

Project id, organization id, billing account id or folder id.
 
For each of those ( except folder id ) we do a call to google cloud to get the corresponding list. 

Once the user has selected that we need to get a list of all the recommenders, which also requires one of the above choices as input parameter.